src/theme/styles/overrides.ts

```diff
@@ -0,0 +1,115 @@
+import { css } from 'styled-components'
+
+// TODO: may not need any of this current CSS for V2
+
+export const web3ModalOverride = css`
```

Comment:
Would it make sense making`overrides` a folder, with one file per "hacked library"
Then, every app, can decide which chunk of CSS they use. I don't think they will all have `web3Modal` for example.

Reply:
mm yes, i agree. i will separate this in another PR

Reply:
added as a subpoint in #1660